src/components/speiling/sykepengsoknader/soknad-felles-oppsummering/Oppsummeringsvisning.tsx

```diff
@@ -16,24 +17,36 @@ const Oppsummeringsvisning: ({
   soknad: { sporsmal },
 }: OppsummeringsvisningProps) => ReactElement = ({
   soknad: { sporsmal },
-}: OppsummeringsvisningProps) => (
-  <>
-    {sporsmal
-      .filter(
-        (sporsmal) =>
-          sporsmal.svar.length > 0 ||
-          sporsmal.undersporsmal.length > 0 ||
-          sporsmal.svartype === SvarTypeDTO.IKKE_RELEVANT
-      )
-      .map((sporsmal) => (
-        <div
-          className="oppsummering__seksjon"
-          key={getKey(sporsmal.tag, sporsmal.id)}
-        >
-          <OppsummeringSporsmal {...sporsmal} />
-        </div>
-      ))}
-  </>
-);
+}: OppsummeringsvisningProps) => {
+  const SummarySection = styled.div`
```

Comment:
Kan evt rename til `OppsummeringSeksjon` for å holde oss på norske komponentnavn her ⛏️

Reply:
Det er nok en god blanding av norske og engelske komponentnavn rundtomkring, så tenkte hovedsakelig for ordens skyld akkurat i denne filen :smile: